tasks.py

```diff
@@ -146,15 +146,15 @@ def print_command(command, env=None):
 
 
 def docker_compose(context, command, **kwargs):
-    """Helper function for running a specific docker-compose command with all appropriate parameters and environment.
+    """Helper function for running a specific docker compose command with all appropriate parameters and environment.
 
     Args:
         context (obj): Used to run specific commands
-        command (str): Command string to append to the "docker-compose ..." command, such as "build", "up", etc.
+        command (str): Command string to append to the "docker compose ..." command, such as "build", "up", etc.
         **kwargs: Passed through to the context.run() call.
     """
     compose_command_tokens = [
-        "docker-compose",
+        "docker compose",
```

Comment:
This is a significant change. We should probably discuss

Reply:
It's been opened many times. It's gone around and around based on how you install Docker and Docker compose.

Reply:
Well, `docker-compose` CLI is EOL as of July - https://docs.docker.com/compose/release-notes/

So it's time to move on to this.

Reply:
I see you're making the same changes in #4287 so whatever we decide let's make sure we are consistent.

Reply:
We need to assert a base Docker version that includes the `docker compose` sub-command and make sure that people are aware of what that is. 

The linked article about Compose V1 being EOL as of July 2023 references Docker Desktop. But what about Linux native Docker users that don't use Docker DesktoP? Does the Docker Engine command also include `compose`? We need these answers and need to make sure they are documented.

Reply:
According to this docs https://github.com/docker/compose, I don't think `compose` is included by default for linux OS.

> ### Where to get Docker Compose

    #### Windows and macOS
    
    Docker Compose is included in [Docker Desktop](https://www.docker.com/products/docker-desktop) for Windows and macOS.
    
    #### Linux
    
    You can download Docker Compose binaries from the [release page](https://github.com/docker/compose/releases) on this repository.
    
    Rename the relevant binary for your OS to docker-compose and copy it to $HOME/.docker/cli-plugins
    
    Or copy it into one of these folders to install it system-wide:
    
    /usr/local/lib/docker/cli-plugins OR /usr/local/libexec/docker/cli-plugins
    /usr/lib/docker/cli-plugins OR /usr/libexec/docker/cli-plugins
    (might require making the downloaded file executable with chmod +x)

Reply:
The [docker ubuntu install docs](https://docs.docker.com/engine/install/ubuntu/) do instruct you to install the docker-compose-plugin package:

`sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin`

Also, the latest deb package of `docker-ce` depends on `docker-ce-cli` which recommends `docker-compose-plugin` so a basic `apt install docker-ce` *should* install the plugin. It doesn't look like this would cause any problems for folks developing for nautobot.